routes/admin/router.js

```diff
@@ -185,57 +185,55 @@ router.delete('/category', function(req, res) {
 router.post('/category/document', function(req, res) {
   if (adminUsername === req.query.username && adminPassword === req.query.password) {
     if (!req.query.id) {
-      res.status(400).json({ error : 'no category id given' }); // TODO TEST
+      res.status(400).json({ error : 'no category id given' }); // TESTED
     } else if (
       (req.body.name == undefined)
       || (req.body.required == undefined)
       || !(typeof req.body.name == "string")
       || !(typeof req.body.required == "boolean")
     ) {
-      res.status(400).json({ error : 'document fields are missing or of the wrong type' }); // TODO TEST
+      res.status(400).json({ error : 'document fields are missing or of the wrong type' }); // TESTED
```

Comment:
It's better to leave them, so we know what is tested without having to look at the tests.